packages/cli/src/commands/billing/add.js

```diff
@@ -1,170 +0,0 @@
-import ansiEscapes from 'ansi-escapes';
-import chalk from 'chalk';
-import ccValidator from 'credit-card';
-import textInput from '../../util/input/text';
```

Comment:
Let's double-check, but I think this might be the only place where this input text module was used as well.

Reply:
It's used in a few other commands as well: `domains`, `dns`, and `teams`.